packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -60,7 +60,7 @@ class UpgradeCommand extends FlutterCommand {
     printStatus('Upgrading engine...');
     code = await runCommandAndStreamOutput(
       <String>[
-        fs.path.join('bin', 'flutter'), '--no-color', 'precache',
+        fs.path.join('bin', 'flutter'), '--no-color', '--no-version-check', 'precache',
```

Comment:
We should pass `--no-version-check` after the command (as below with the `doctor` call), or move the one for doctor to before the command, for consistency.

Reply:
I've moved the doctor one forward; seems more obvious that they're global args if they come before the command.